HiP-DataStore.Model/Rest/ExhibitStatisticResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Rest
+{
+    public class ExhibitStatisticResult
```

Comment:
What exactly do Year,Month and Day stand for? Documentation would make that clear 😉